index.html

```diff
@@ -3340,6 +3358,11 @@ <h2>
         <h2>
           Payment Handler Matching
         </h2>
+        <p>
+          A user agent's payment handler matching algorithm takes into account
```

Comment:
I feel like "the requirements of this specification" needs to link to something - as the specification has a lot of requirements and it's not clear exactly which we are talking about here. 

Also, there is no defined "payment handler matching algorithm", so maybe this text should not be here at all.

Reply:
@marcoscaceres,

Here is some additional detail. If you want to understand why some apps show up in the list and some don't, you need to be aware of:
 * Payment method identifier matching defined in PR API.
 * canMakePayment() matching, which is payment method specific 
 * payment method manifest matching
 * potentially other matching to come (e.g., if we add a layer for 3DS flows where the merchant can say "I only accept payments from apps that do 3DS flows given the jurisdiction in which I operate.")

I'd like to raise awareness of this. I welcome your suggestions of how and where that might be done.

Ian

Reply:
We can do it here, but as a note (i.e., class = "informative" and in `<div class=note>`)... like:

### How user agents match payment handlers
A merchant can accept payment by supporting many different payment handlers (e.g, "basic-card" and "`https://example.com/pay`"). As such, when a merchant initiates a request for payment, a user agent will typically display a "payment sheet" and give the user the option to select from the matching payments handlers available. The matching is done on: 
  
  * The payment method identifier: e.g., the user agent supports "basic-card", 
     but not "https://example.com/pay". So "basic-card" would be the only 
     option available to the user. 

  * User preference - a user might say they exclusively want to use "https://example.com/pay", and not be bothered with "basic-card". So the use completes the payment with the "https://example.com/pay" payment method. 

And so on... 

I think it would be overstepping to mention payment method manifest or 3D at this point (obviously). But if those two could be generalized, then we should include them above maybe... depends if it would imply support for something none supports yet.